tests/integration/domains/osu_test.py

```diff
@@ -0,0 +1,212 @@
+from __future__ import annotations
+
+import hashlib
+import secrets
+from datetime import datetime
+from uuid import UUID
+
+from fastapi import status
+from httpx import AsyncClient
+
+from app import encryption
+
+
+async def test_score_submission(http_client: AsyncClient) -> None:
```

Comment:
This is currently making real 3rd party (e.g. osu! api) calls in CI.

Probably not ideal -- would prefer to mock out 3rd party calls to reduce test flakiness & abuse, though it makes for quite the good integration test.

Reply:
Not 100% sure how I would ideally resolve this -- perhaps we could have some test data in the repo, such as .osu files, etc. such that we can continue to itest the majority of the code flows, but without the 3rd party network calls.

Reply:
Nvm -- resolving it in this PR here: https://github.com/osuAkatsuki/bancho.py/pull/508/commits/cc07c5d00147156449bf3e6e6cec8ecad8c7105d